apps/tailwind-components/components/input/CheckboxGroup.vue

```diff
@@ -1,21 +1,28 @@
 <template>
-  <div :id="`${id}-checkbox-group`">
+  <div :id="`${id}-checkbox-group`" :aria-describedby="describedBy">
```

Comment:
I think invalid styling can be placed here. I would probably add a border around the group of checkboxes as opposed to each checkbox. Placing the error at the checkbox icon level would indicate that there is an issue with that particular option, when the issue is occurs at the group level.

Reply:
most other frameworks don't do that because they only highlight user editable items.